fh-sync.d.ts

```diff
@@ -0,0 +1,350 @@
+// Type definitions for fh-sync
+// Project: https://github.com/feedhenry/fh-sync
+// Maintainer feedhenry-dev@redhat.com
+
+declare module SyncCloud {
+
+      /**
+       * Valid actions (the "fn" param) that can be passed to sync.invoke
+       */
+      type InvokeAction = 'sync'|'syncRecords'|'listCollisions'|'removeCollision'
```

Comment:
Using this will allow TS to flag if someone has a typo or wrong action in their `sync.invoke` options